### PR TITLE
Add spanset to dictionary

### PR DESCRIPTION
**Pull request by jdbaldry**
Defined in https://github.com/grafana/website/pull/18319

### CODE REVIEW

File: vale/dictionaries/en_US-grafana.dic [hunk 1]
```diff
@@ -1,4 +1,4 @@
-85
+86
```
Comment (clayton-cornell) [inline vale/dictionaries/en_US-grafana.dic:1]: I'm interested in this value. I've tried poking the Vale docs, but not sure what to search for. I can see it's a count of entries in this file.. but for what purpose? What happens if we forget to increment this value?

**Review by clayton-cornell — APPROVED** [review]

File: vale/dictionaries/en_US-grafana.dic (hunk 1 above)
Comment (jdbaldry) [reply vale/dictionaries/en_US-grafana.dic:1]: Ah yeah, this is not actually a Vale thing but rather an aspect of the Hunspell dictionary format. 

The best resource I have right now is https://manpages.ubuntu.com/manpages/focal/man5/hunspell.5.html.

The need to count the entries is actually why I have a separate `.wordlist` file that we can just add entries to and then run `make all` which generates the `.dic` file from the `.wordlist` file so the number is always right.

**Review by knylander-grafana — APPROVED** [review]
Thank you for adding this term, Jack!

**Comment by jdbaldry** [comment]
What do you think about adding it to the [glossary](https://github.com/grafana/website/blob/master/data/glossary.yaml?rgh-link-date=2024-02-15T10%3A08%3A22Z)?

It would be useful for this PR to be able to reference that as justification for the word being in the dictionary.

**Comment by knylander-grafana** [comment]
> What do you think about adding it to the [glossary](https://github.com/grafana/website/blob/master/data/glossary.yaml?rgh-link-date=2024-02-15T17%3A33%3A53Z)?
> 
> It would be useful for this PR to be able to reference that as justification for the word being in the dictionary.

Sure. We don't currently have spanset created as an entry. I'll see if we have an existing definition some where. j

**Comment by jdbaldry** [comment]
Awesome, thank you!

**Pull request merged**